<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kütüphane Otomasyonu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>550</height>
     </rect>
    </property>
    <widget class="QWidget" name="tabMain">
     <attribute name="title">
      <string>Ana Menü</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabBooks">
     <attribute name="title">
      <string>Kitaplar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetBooks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>750</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBookID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBookTitle">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxBookCategory">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxBookAuthor">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddBook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdateBook">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteBook">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSearchBook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Search Books</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSearchBook">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCategories">
     <attribute name="title">
      <string>Kategoriler</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetCategories">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>750</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCategoryID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCategoryName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddCategory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdateCategory">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteCategory">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAuthors">
     <attribute name="title">
      <string>Yazarlar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetAuthors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>750</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAuthorID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAuthorName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddAuthor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdateAuthor">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDeleteAuthor">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelAuthor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignVCenter</set>
    </property>
    <property name="text">
     <string>Alperen KOÇYİĞİT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
